jobs/v3/howto/auto_complete_sample.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package sample
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	talent "google.golang.org/api/jobs/v3"
+)
+
+// [START auto_complete_job_title]
+
+// jobTitleAutoComplete suggests the job titles of the given companyName based on query.
+func jobTitleAutoComplete(service *talent.Service, parent string, companyName string, query string) (*talent.CompleteQueryResponse, error) {
+	complete := service.Projects.Complete(parent).Query(query).LanguageCode("en-US").Type("JOB_TITLE").PageSize(10)
+	if companyName != "" {
+		complete.CompanyName(companyName)
+	}
+	resp, err := complete.Do()
+	if err != nil {
+		log.Fatalf("Failed to auto complete with query %s in company %s, Err: %v", query, companyName, err)
```

Comment:
Return an error here. In Go, it's common to give context to the error, then a colon (:), then the error. For example:

```
return fmt.Errorf("failed to auto complete with query %q in company %q: %v", query, companyName, err)
```

Note: The first letter is not capitalized (https://github.com/golang/go/wiki/CodeReviewComments#error-strings) and the query & company name are printed with `%q` so they're quoted. Also, I think the only return value should be the error (not the response) - see below.